data/categories.txt

```diff
@@ -836,7 +836,7 @@ es:3Restaurante|3café
 eu:3Jatetxea|3Kafetegia
 sv:3Restaurang|3café
 th:3ร้านอาหาร
-tr:3Restoran|3kafeterya
+tr:2Restoran|3kafeterya|2lokanta|2kafe
```

Comment:
Why you decided to use 2? It is usually a very low number, considering that restaurants are already searched by "Where to eat" shortcut.